backend/schema/corpora_schema.md

```diff
@@ -1,125 +1,71 @@
-    of one-to-many mappings between gene symbols and other gene ids. In cases where there are duplicated feature identifiers, they will be summed during
-    curation.
-*   **Type annotations or inferrable types**. For both expression values and metadata, we don't want to have to guess at the type. Typing is already explicit
-    in anndata, loom, Seurat and Bioconductor formats.
+    of one-to-many mappings between gene symbols and other gene ids. In cases where there are duplicated feature identifiers, they will need to be appropriately
+    combined before submission. For example, raw counts will be summed and logged counts will be exponentiated, summed, and logged.
 *   **No PII**. No metadata can be personally identifiable: no names, dates of birth, specific locations, etc. There's a
     [list](https://docs.google.com/document/d/1nlUuRiQ8Awo_QsiVFi8OWdhGONjeaXE32z5GblVCfcI/edit?usp=sharing).
 
-
-#### Matrix Layers
+### Matrix Layers
 
 The count matrix itself can exist in several forms. Generally, there is a "raw" count matrix without scaling, filtering, normalization, etc. And there is
 a "final" matrix that is used in the publication with QC and various corrections. There can also be several intermediate matrices.
 
-These different transformations of the count matrix are called "layers" in several file formats. Submissions to Corpora must contain at least one layer,
-the "raw" layer. They may optionally also have a "final" layer used for presentation, and each layer must be identified.
-
-
-### Remix Metadata
+These different transformations of the count matrix are called "layers" in several file formats. Submissions to the Data Portal must contain the "raw"
+layer.
 
-The remaining part of the schema defines what's needed for the Corpora remix. The words "must" and "requirement" are used throughout, but note the
-discussion above regarding curation policies as well the [caution against](#Software-vs-policy) introducing any
-software dependencies on these fields.
 
+### Schema Version
 
-#### Schema Version
-
-Remixed Corpora datasets must store the version of the schema they follow (that is, the version of this document) as
-well as the version of the particular encoding used:
+Datasets in the Data Portal must store the version of the schema they follow (that is, the version of this document) as
+well as the version of the particular encoding used. The encoding is documented
+[elsewhere](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema_h5ad_implementation.md) and describes techincal details
+of how the schema should be serialized in a particular file format.
 
 **Field name**|**Constraints**
 :--|:--
 corpora_schema_version|Follows [SemVer](https://semver.org/) conventions
 corpora_encoding_version|Follows [SemVer](https://semver.org/) conventions
 
-#### Integration Metadata
+### Integration Metadata
 
 To support data integration, each cell must have the following metadata:
```

Comment:
For organoids or cell cultures, is it still required that each cell have cell_type field filled?

Reply:
I would say no (and I haven't been including it in the few cell culture datasets we have). Does that seem reasonable to you?